### PR TITLE
Basic appendix sweep2 2

### PR DESCRIPTION
**Pull request by Impakt**

### CODE REVIEW

File: settingup.tex [hunk 1]
```diff
@@ -108,8 +108,8 @@ \subsection{Replacing the Real-Time Clock Battery}
 battery socket \textit{away} from the battery to remove it. Next, insert the new battery with the side labelled
 {\bf + facing up}, and press it into place.
 
-Once you have re-assembled your MEGA65, you can set the time in the Configure menu. You can read more about setting
-the time on page \pageref{configuring-chipset}.
+Once you have re-assembled your MEGA65, you can set the time in the Configure menu. For more information on how to set
+the Real-Time Clock, refer to the Configuring Utility section on page \pageref{sec:configuration-utility}.
```
Comment (gurcei) [inline settingup.tex:112]: typo, should be: ...refer to the **_Configuration_** Utility...

**Review by gurcei — APPROVED** [review]
Just a few obs here and there :)

**Review by gurcei — CHANGES_REQUESTED** [review]
Oops, I should hold off on the approve until you work your way through the comments :)

File: appendix-basic65.tex [hunk 2]
```diff
@@ -3447,7 +3456,7 @@ \subsection{ERASE}
    between erasing and recovery, which may have altered the
    contents of the file.
 
-\item [Remarks:] {\bf ERASE filename} works similarly to {\bf SCRATCH filename}.
+\item [Remarks:] {\bf ERASE} works similarly to {\bf SCRATCH}.
```
Comment (Edilbert) [inline appendix-basic65.tex:3459]: ERASE doesn't work similar, but identical to SCRATCH.
It's an alias and calls the same routine.
DELETE filename
ERASE filename
SCRATCH filename
are different names for the same action.

File: appendix-basic65.tex [hunk 3]
```diff
@@ -4014,7 +4025,7 @@ \subsection{FRE}
                {\bf FRE(1)} returns the number of free bytes in
                bank 1, which is the bank for BASIC variables, arrays
                and strings. {\bf FRE(1)} also triggers
-               "garbage collection", which is a process that collects
+               ``garbage collection'', which is a process that collects
                used strings at the top of the bank, thereby
```
Comment (Edilbert) [inline appendix-basic65.tex:4029]: "strings in use" describes the status better, than "used strings"

File: appendix-basic65.tex [hunk 4]
```diff
@@ -1928,8 +1928,9 @@ \subsection{COLOR}
 \index{BASIC 65 Commands!COLOR}
 \begin{description}[leftmargin=2cm,style=nextline]
 \item [Token:] \$E7
-\item [Format:] {\bf COLOR} <{\bf ON} | {\bf OFF}>
-\item [Usage:] Enables or disables
+\item [Format:] {\bf COLOR} <{\bf ON} | {\bf OFF}>\\
+                {\bf COLOR} colour
```
[the remark this answers is not in the document]
Comment (Impakt) [reply appendix-basic65.tex:1932]: done

File: appendix-basic65.tex [hunk 5]
```diff
@@ -2979,10 +2986,12 @@ \subsection{DT\$}
 \item [Format:] {\bf DT\$}
 \item [Usage:]  {\bf DT\$} holds the current date and is updated before
                 each usage from the RTC (Real-Time Clock).
-                The RTC can be set in the Configure Menu.
                 The string {\bf DT\$} is formatted as:
                 "DD-MON-YYYY", for example: "04-APR-2021".
 \item[Remarks:] {\bf DT\$} is a reserved system variable.
+                For more information on how to set the Real-Time Clock,
+                refer to the Configuring Utility section on page
```
[the remark this answers is not in the document]
Comment (Impakt) [reply appendix-basic65.tex:2993]: good find, fixed

File: appendix-basic65.tex [hunk 6]
```diff
@@ -2953,7 +2961,6 @@ \subsection{DSAVE}
    \unitdefinition
 
 \item [Remarks:]
-   The {\bf DVERIFY} can be used after {\bf DSAVE} to check,
    if the saved program on disk is identical to the program
```
[the remark this answers is not in the document]
Comment (Impakt) [reply appendix-basic65.tex:2964]: fixed. obviously i was getting tired!

File: appendix-basic65.tex (hunk 5 above)
[the remark this answers is not in the document]
Comment (Impakt) [reply appendix-basic65.tex:2993]: i copied/pasted that snippet from another page which also had the wrong wording. Fixed everywhere now

File: appendix-basic65.tex [hunk 7]
```diff
@@ -3040,15 +3049,15 @@ \subsection{EDIT}
 \begin{description}[leftmargin=2cm,style=nextline]
 \item [Format:] {\bf EDIT} <{\bf ON} | {\bf OFF}>
 
-\item [Usage:]  {\bf EDIT} switches the builtin editor
-               either to text mode {\bf EDIT ON}
-               or BASIC program editor {\bf EDIT OFF}.
+\item [Usage:]  {\bf EDIT} switches the built-in editor
+               either to text mode with {\bf EDIT ON},
+               or ot the BASIC program editor with{\bf EDIT OFF}.
```
[the remark this answers is not in the document]
Comment (Impakt) [reply appendix-basic65.tex:3054]: fixed

File: appendix-basic65.tex [hunk 8]
```diff
@@ -3059,18 +3068,18 @@ \subsection{EDIT}
...
 
                The mode of the editor can be recognised by looking at the prompt:
-               In program mode, the prompt is: \screentext{READY.}, whilst in text mode
-               the prompt is: {\bf OK}.
+               In program mode, the prompt is \screentext{READY.}, whilst in text mode
+               the prompt is \screentext{OK}.
 
-               The text mode affects entered lines with leading number only,
-               lines with no linenumber are executed as BASIC commands,
+               The text mode affects entered lines with leading numbers only,
```
[the remark this answers is not in the document]
Comment (Impakt) [reply appendix-basic65.tex:3081]: I agree, fixed

File: appendix-basic65.tex [hunk 9]
```diff
@@ -3135,17 +3144,17 @@ \subsection{EDMA}
...
+   {\bf command} 0: copy, 1: mix, 2: swap, 3: fill.
 
    {\bf length} number of bytes (maximum = 65535).
 
    {\bf source}  28-bit address of read area or fill byte.
 
    {\bf target} 28-bit address of write area.
 
-   {\bf sub} sub command (see chapter on DMA controller).
+   {\bf sub} sub command (see chapter on DMA controller in the MEGA65 Complete Compendium).
```
[the remark this answers is not in the document]
Comment (Impakt) [reply appendix-basic65.tex:3155]: reverted

File: appendix-basic65.tex [hunk 10]
```diff
@@ -3989,7 +3999,8 @@ \subsection{FOREGROUND}
                range of 0 to 31.
                Refer to the table under
                {\bf BACKGROUND} on page \pageref{colourtable}
-               for the {\bf colour} values and their corresponding colours.
+               for the colour values and their corresponding colours.
+\item [Remarks:] {\bf FOREGROUND} is a synonym of {\bf COLOR}.
```
[the remark this answers is not in the document]
Comment (Impakt) [reply appendix-basic65.tex:4003]: `{\bf COLOR} also has the ability to change the foreground colour.`?

File: appendix-basic65.tex [hunk 11]
```diff
@@ -4977,7 +4989,7 @@ \subsection{KEY}
 \item [Remarks:] The sum of the lengths of all assigned strings
                  must not exceed 240 characters.
                  Special characters such as RETURN or QUOTE are entered
-                 using their codes with the CHR\$(code) function.
+                 using their codes with the {\bf CHR\$} command.
```
[the remark this answers is not in the document]
Comment (Impakt) [reply appendix-basic65.tex:4992]: fixed, and fixed a couple other instances that referred to is as a command as well for bonus points

File: settingup.tex (hunk 1 above)
Comment (Impakt) [reply settingup.tex:112]: fixed

File: appendix-basic65.tex (hunk 2 above)
Comment (Impakt) [reply appendix-basic65.tex:3459]: updated them all to say `is a synonym for`

File: appendix-basic65.tex (hunk 3 above)
Comment (Impakt) [reply appendix-basic65.tex:4029]: fixed

File: appendix-basic65.tex [hunk 12]
```diff
@@ -1940,6 +1945,11 @@ \subsection{COLOR}
                for screen characters. This speeds up screen
                handling, which could be useful when moving characters with different colours is
                not required.
+
+
+\item
+\item which is documented
+               on page \pageref{foregroundcommand}.
```
Comment (gurcei) [inline appendix-basic65.tex:1952]: Not sure what's up with this section. Seems loosely dangling in the middle of newhere. Was it a cut/paste misfire?

File: appendix-basic65.tex [hunk 13]
```diff
@@ -1940,6 +1945,11 @@ \subsection{COLOR}
...
 \item [Example:] \screentext{COLOR ON} - with colour/attribute handling \\
```
Comment (gurcei) [inline appendix-basic65.tex:1953]: Maye we can add an example of the colour index usage too, e.g.:
```
\screentext{COLOR 2 : REM SET TEXT COLOUR TO RED}
```

File: appendix-basic65.tex (hunk 10 above)
[the remark this answers is not in the document]
Comment (gurcei) [reply appendix-basic65.tex]: Yep, that's fine my me

File: appendix-basic65.tex (hunk 12 above)
Comment (Impakt) [reply appendix-basic65.tex:1952]: yeah looks like it, removing it now

File: appendix-basic65.tex (hunk 13 above)
Comment (gurcei) [reply appendix-basic65.tex]: Aah, saw that you turned it into a program, so I tried the the program, and the 2nd line `COLOR OFF` had the side-effect of dismissing the 3rd line's attempt to change to red.

Maybe it'd be best to just have them like they were before, direct lines, but with explanations to the right, e.g.:
```
\screentext{COLOR 2} - set text colour to red \\
\screentext{COLOR ON} - with colour/attribute handling \\
\screentext{COLOR OFF} - no colour/attribute handling
```

Just for insight, here's a more elaborate example showing the COLOR ON/OFF behaviour:
image

**Review by gurcei — APPROVED** [review]
all good, thx :)

**Pull request merged**